src/Icons/src/Iconify.php

```diff
@@ -95,6 +95,29 @@ public function fetchSvg(string $prefix, string $name): string
         return $content;
     }
 
+    /**
+     * @internal
+     */
+    public function getIconSets(): array
+    {
+        return $this->sets()->getArrayCopy();
+    }
+
+    /**
+     * @internal
+     */
+    public function searchIcons(string $prefix, string $query)
```

Comment:
```suggestion
    public function searchIcons(string $prefix, string $query)
```

Entire class is internal currently.

Reply:
Thanks!